### PR TITLE
[plugin.video.reddit_viewer] 3.3.8

### PR DESCRIPTION
**Pull request by gedisony**
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.reddit_viewer/resources/lib/guis.py [hunk 1]
```diff
@@ -51,26 +54,45 @@ class cGUI(xbmcgui.WindowXML):
...
+        #if self.plot_font:
+        #    self.ctl_plot_textbox = self.getControl(self.CONTROL_ID_FOR_PLOT_TEXTBOX)
+        #    self.ctl_plot_textbox.setLabel('Status', 'font14', '0xFFFFFFFF', '0xFFFF3300', '0xFF000000')
+
+        #url="plugin://plugin.video.reddit_viewer/?url=plugin%3A%2F%2Fplugin.video.youtube%2Fplay%2F%3Fvideo_id%3D73lsIXzBar0&mode=playVideo"
+        #url="http://i.imgur.com/ARdeL4F.mp4"
         if self.include_parent_directory_entry:
             back_image='DefaultFolderBackSquare.png'
-            listitem = xbmcgui.ListItem(label='..')
+            listitem = xbmcgui.ListItem(label='..', label2="", iconImage=back_image)
```
Comment (enen92) [inline plugin.video.reddit_viewer/resources/lib/guis.py]: same

File: plugin.video.reddit_viewer/resources/lib/utils.py [hunk 2]
```diff
@@ -833,142 +434,141 @@ def xbmcVersion():
...
-            #xbmc.executebuiltin("StartAndroidActivity(org.mozilla.firefox,android.intent.action.VIEW,,"+url+")")
-            # ___ Open media with Chrome
-            #xbmc.executebuiltin("StartAndroidActivity(com.android.chrome,,,"+url+")")
+
+def addDir(name, url, mode, iconimage, type_="", listitem_infolabel=None, label2=""):
+
+    u = sys.argv[0]+"?url="+urllib.quote_plus(url)+"&mode="+str(mode)+"&type="+str(type_)
+
+    ok = True
+    liz = xbmcgui.ListItem(label=name, label2=label2, iconImage="", thumbnailImage='')
```
Comment (enen92) [inline plugin.video.reddit_viewer/resources/lib/utils.py]: same

**Comment by gedisony** [comment]
I made the requested changes. 
Thanks btw for suggesting that I move everything out of default.py. It really improved the speed.

**Comment by enen92** [comment]
Thanks 👍

**Pull request merged**